projects/centre_console/src/fault_monitor.c

```diff
@@ -0,0 +1,50 @@
+#include "fault_monitor.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "centre_console_events.h"
+#include "centre_console_fault_reason.h"
+#include "power_fsm.h"
+#include "status.h"
+#include "watchdog.h"
+
+#define NO_FAULT 0
+
+typedef struct FaultMonitorStorage {
+  uint16_t bps_fault_bitset;
+  WatchdogStorage watchdog_storage;
+} FaultMonitorStorage;
```

Comment:
Shouldn't this be in the header, also what's the point of storing the fault_bitset?

Reply:
True we don't do anything with the bitset, so I just made it a static watchdog storage